client/landing/jetpack-cloud/routes.js

```diff
@@ -6,24 +6,36 @@ import page from 'page';
 /**
  * Internal dependencies
  */
-import {
-	clientRender,
-	jetpackCloud,
-	jetpackBackups,
-	jetpackScan,
-	makeLayout,
-	setupSidebar,
-} from './controller';
 import { normalize } from 'lib/route';
+import { siteSelection } from 'my-sites/controller';
+import { clientRender, makeLayout, setupSidebar } from './controller';
+import { jetpackCloudDashboard } from './sections/dashboard/controller';
+import { jetpackCloudBackups } from './sections/backups/controller';
+import { jetpackCloudBackupDetail } from './sections/backup-detail/controller';
+import { jetpackCloudBackupDownload } from './sections/backup-download/controller';
+import { jetpackCloudBackupRestore } from './sections/backup-restore/controller';
+import { jetpackCloudScan } from './sections/scan/controller';
+import { jetpackCloudScanHistory } from './sections/scan-history/controller';
+import { jetpackCloudSettings } from './sections/settings/controller';
 
 const router = () => {
 	page( '*', normalize );
 
-	page( '/', setupSidebar, jetpackCloud, makeLayout, clientRender );
+	page( '/', setupSidebar, jetpackCloudDashboard, makeLayout, clientRender );
 
-	page( '/backups', setupSidebar, jetpackBackups, makeLayout, clientRender );
+	page( '/backups', siteSelection, setupSidebar, jetpackCloudBackups, makeLayout, clientRender );
+	page( '/backups/:site', siteSelection, setupSidebar, jetpackCloudBackups, makeLayout, clientRender );
+	page( '/backups/:site/detail/:backupId', siteSelection, setupSidebar, jetpackCloudBackupDetail, makeLayout, clientRender );
```

Comment:
What do you think about having the `:site` part after the subpage, i.e.:

```
/backups/detail/:site/:backupId
```

I don't have any strong opinion on either approach. However, it seems that on wordpress.com we are using a structure like this as well:

```
/me/purchases/:site/:purchaseId
/stats/month/:site?startDate=2019-7-1
```

Probably one of the advantages of such a structure is a slightly easier way of defining links. E.g. in the sidebar we would have something like this:

```js
link={ '/backups/restore' + this.props.siteSuffix }
```

This is obviously not a blocker of any kind, just something worth a second thought. What is your opinion?

Reply:
Interesting! I have no strong opinion, only looking to follow convention. I'll consider changing it. Thanks!